src/platform/lumin-runtime/types/controller-events.js

```diff
@@ -0,0 +1,4 @@
+export const ControllerEvents = {
+  'onEvent': { subName: 'onEvent' },
+  'onUpdateLoop': { subName: 'onUpdate' }
```

Comment:
I'm not a fan of the name of the property being exposed.  Sadly I cannot provide a better alternative

Reply:
This is just a map so we can change the name anytime we won't in case we need to.

Reply:
Renamed !